docs/configuration/registration/objects.rst

```diff
@@ -159,91 +287,96 @@ Objects API. The top-level has meta-data about the form submission, and the
            }
          }
 
+Using the *Variables mapping (v2)* configuration
+================================================
 
-To configure the Objects API follow these steps:
+When using the variables mapping configuration, each variable can be mapped to a specific location in the object type
```

Comment:
I think above you went for `objecttype` without space, I am not sure which one is the preferred version, but we should maybe stick to 1 of the two

Reply:
Yeah I'm not sure, the name of the API is _Objecttypes API_, but I find it better to refer to _an object type_ with a space

Reply:
Spelling-wise, a space is needed, but I've chosen to stick to the official naming of the product (without space).